packages/ui-components/src/components/dropdown-list-item/dropdown-list-item.tsx

```diff
@@ -0,0 +1,44 @@
+import { Component, Host, h, Prop, EventEmitter, Event } from '@stencil/core';
+import { EOtherIconName } from '../icon/icon.types';
+import { IDropdownListItem, IDropdownListItemEvents } from './dropdown-list-item.types';
+
+@Component({
+	tag: 'kv-dropdown-list-item',
+	styleUrl: 'dropdown-list-item.scss',
+	shadow: true
+})
+export class KvDropdownListItem implements IDropdownListItem, IDropdownListItemEvents {
+	/** (required) The text to display on the item */
+	@Prop({ reflect: true }) label!: string;
+	/** (required) The item value */
+	@Prop({ reflect: true }) value!: string;
+	/** (optional) If `true` the item is selected */
+	@Prop({ reflect: true }) selected?: boolean = false;
+	/** (optional)  If `true` the item is togglable */
+	@Prop({ reflect: true }) togglable?: boolean = false;
+
+	/** Emitted when the user clicks on the item */
+	@Event() itemSelected: EventEmitter<string>;
+
+	private onItemClick = () => {
+		this.selected = !this.selected;
+		this.itemSelected.emit(this.value);
+	};
+
+	render() {
+		return (
+			<Host>
+				<div
+					class={{
+						'dropdown-list-item': true,
+						'selected': this.selected
+					}}
+					onClick={this.onItemClick}
+				>
+					{this.togglable && <kv-icon name={this.selected ? EOtherIconName.IconCheckState : EOtherIconName.IconUncheckState} custom-class="icon-16"></kv-icon>}
+					<div class="item-label">{this.label}</div>
```

Comment:
Can we have a slot here? For example to put a circle status before on the label, or have other icon, or to draw the assets hierarchy path?

Reply:
Right now we only have the text-field dropdown on zeplin, let's add functionality as needed.